.env.sample

```diff
@@ -0,0 +1,4 @@
+FLASK_DEBUG=True
+TENANT_NAME=<tenant id>
```

Comment:
I noticed that you changed it from `TENANT_ID` to `TENANT_NAME` in your latest commit, probably influenced by `app_config_b2c.py`'s accepting a short name such as "contoso".

`app_config_b2c.py`'s choice predated Azure Portal's hydrating this template into a run-able sample. Now, on a second thought, this template should probably stay with expecting a full tenant id such as "contoso.onmicrosoft.com", because that way it would also be compatible with an input of tenant guid.

Also, I doubt either "tenant name" or "tenant id" is an established concept, so, we better also provide an example value in comments.
```suggestion
# Expects a full tenant name such as "contoso.onmicrosoft.com", or its GUID
TENANT_ID=<tenant id>
```

Would you mind changing `.env.sample.b2c`, `app_config.py` and `app_config_b2c.py` accordingly?

Reply:
Ahh okay, got it, can do.

Reply:
Just triple-checking, that full tenant name is compatible with this string? I don't see the full tenant name in the portal overview, just the (short) display name and tenant ID. 

authority_template = "https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{user_flow}"

Reply:
> I don't see the full tenant name in the portal overview, just the (short) display name and tenant ID.

Which portal overview? I only know that Azure Portal's quickstart will hydrate this template for an AAD app, and since [our preexisting `app_config.py` kind of expecting `https://login.microsoftonline.com/full_tenant_name`](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/91/files#diff-89923134625b190c45892998f54559a1a4d2f4733d3c33c85344a6edf498899aL13-L14), I believe Azure Portal's QuickStart uses full tenant name.

> authority_template = "https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{user_flow}"

That line is from the `app_config_b2c.py`, so, yeah, it would need to be changed into `authority_template = "https://{tenant}.b2clogin.com/{tenant_id}/{user_flow}"`

Reply:
I mean the portal page for App registrations, I see where to get the GUID version of the Tenant ID, but not the "full name" version, so I'm wondering if the comment can just refer to the GUID version?

<img width="1095" alt="Screenshot 2023-02-28 at 3 38 40 PM" src="https://user-images.githubusercontent.com/297042/222007972-2fe12349-91b7-42c0-8d4c-7041a4735b75.png">

Reply:
Good question.

The "full name" is shown in the Domain column in the ["directories + subscriptions" selection page](https://portal.azure.com/#settings/directory).

Reply:
Ah I see it there, thanks. I think I've made the desired change here and in the article PRs, btw.